play/src/front/Chat/Components/UserList/UserActionButton.svelte

```diff
@@ -41,39 +42,81 @@
         }
         closeChatUserMenu();
     };
+
+    let loadingDirectRoomAccess = false;
+
+    const openChat = async () => {
+        let room: ChatRoom | undefined = chatConnection.getDirectRoomFor(user.chatId);
+        if (!room)
+            try {
+                loadingDirectRoomAccess = true;
+                room = await chatConnection.createDirectRoom(user.chatId);
+            } catch (error) {
+                console.error(error);
```

Comment:
Add a Sentry log here.

Note: we should start thinking at a global way to handle errors that adds the error in the console, logs it in Sentry AND displays an error message to the user.